tennis-scheduler-web-app/src/components/Login.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react"
+import { Link } from 'react-router-dom'   
+
+export const Login = () => {
+
+    const [email, setEmail] = useState('')
+    const [pass, setPassword] = useState('')
+
+    const loginSubmit = (e) => {
```

Comment:
When having only one argument brackets can be omitted, it is not mandatory but still a nice feature :sparkles:.
const loginSubmit = e => {...}